pageComponents/NewPostPage/PlayerHandSelectors.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import styled from "styled-components";
+import SelectablePlayingCard from "../../components/SelectablePlayingCard";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PlayingCard, { Rank, Suit } from "../../models/PlayingCard";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  playerIndexes: number[];
+  playerHands: { left: PlayingCard | null; right: PlayingCard | null }[];
```

Comment:
I think we can use tuple for this kind of situation

```suggestion
  playerHands: [PlayingCard | null, PlayingCard | null][];
```